### PR TITLE
chore: update documentation to include tooling and usage instructions

### PR DESCRIPTION
**Pull request by austin94**
## Description

This PR updates the `README` documentation to include more detailed usage instructions. The usage categories of `Utility Classes` and `Class Composition` are both used to style the React example.

I also included an example of the Tailwind IntelliSense VS Code plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,37 @@ module.exports = {
...
+### Utility Classes
+
+The simplest way to apply Garden Design Tokens is by using the
+[Tailwind utility classes](https://tailwindcss.com/docs/utility-first).
+
+```html
+<img
+  class="rounded-full h-8 w-8 border border-green-600"
+  src="avatar.png"
+  alt="Avatar image" />
```
Comment (hzhu) [inline README.md:43]: nit: Should the word "image" be removed to discourage redundant words read aloud on screen readers? I've seen the [img-redundant-alt](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/img-redundant-alt.md#img-redundant-alt) rule used in Garden and have come across that lint warning before.

Comment (austin94) [reply README.md]: Good catch. Removed.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,37 @@ module.exports = {
...
+  alt="Avatar" />
+```
+
+To reduce the bundle size impact of these utilities consider using the [PurgeCSS functionality](https://tailwindcss.com/docs/controlling-file-size#removing-unused-css)
+provided within Tailwind.
+
+### Class Composition with `@apply`
+
+Tailwind provides several [PostCSS directives](https://tailwindcss.com/docs/functions-and-directives/)
+to consume utility styles. The [@apply directive](https://tailwindcss.com/docs/functions-and-directives/#apply)
```
Comment (jzempel) [inline README.md:52]: "...~to consume~ for inserting utility styles into CSS."

Comment (austin94) [reply README.md:52]: Updated.

**Pull request merged**